src/components/UserSignUp.jsx

```diff
@@ -21,22 +22,35 @@ const UserSignUp = () => {
       });
   };
   return (
-    <div className="sign-up-container">
-      <form onSubmit={handleSignUp}>
-        <h1>Create Account</h1>
-        <input
-          type="email"
-          placeholder="Enter Your Email"
-          value={email}
-          onChange={(e) => setEmail(e.target.value)}></input>
-        <input
-          type="password"
-          placeholder="Enter Your Password"
-          value={password}
-          onChange={(e) => setPassword(e.target.value)}></input>
-        <button type="submit">Sign Up</button>
-      </form>
-    </div>
+    <section className="max-w-[60rem]">
+      <Card className="sign-up-container p-6 m-3">
+        <form onSubmit={handleSignUp}>
```

Comment:
I think we don't need form here. If we remove this, we can move handleSignup function to `<Button onClick={handleSignUp}>` and we can also remove `e.preventDefault();` in handleSignUp